app/scripts/modules/core/src/deploymentStrategy/strategies/rollingredblack/additionalFields.html

```diff
@@ -1,9 +1,21 @@
-<div class="form-group">
+<div class="form-group" style="margin-top: 15px">
```

Comment:
We do this in some of the other strategies ... The label looked a little squishy otherwise. 

Not necessarily a fan of the inline style but 'twas there previously.

Reply:
We do this in a lot of places, nothing to worry about